frontend/web/src/app/components/customer-list/customer-list.component.ts

```diff
@@ -100,10 +101,10 @@ export class CustomerListComponent implements OnInit, AfterViewInit {
   }
 
   private prepareTriggers() {
-    /*let _self = this;
+    let _self = this;
     (<any>this.table).on('click-row.bs.table', function (row, $element) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Type assertion using the '<>' syntax is forbidden. Use the 'as' syntax instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS4137)